test/AngularSpec.js

```diff
@@ -1861,6 +1861,44 @@ describe('angular', function() {
     });
   });
 
+  describe('isError', function() {
+    it('should not assume objects are errors', function() {
+      expect(isError({ message: 'A fake error', stack: 'no stack here'}))
+        .toBe(false);
+    });
+
+    it('should detect simple error instances', function() {
+      expect(isError(new Error())).toBe(true);
+    });
+
+    function testError(createError) {
```

Comment:
it's specific to the actual call to the testErrorFromDifferentContext function. Like the lifetime of the iframe is shorter than a test, no point leaving it hanging around until afterAll/beforeAll is called.